stablehlo/dialect/TypeInference.h

```diff
@@ -313,7 +313,7 @@ LogicalResult inferTransposeOp(Optional<Location> loc, Value operand,
 
 LogicalResult inferTriangularSolveOp(
     Optional<Location> location, Value a, Value b, bool leftSide,
-    bool isTransposeAInvalid,
+    bool isTransposeAInvalid, bool isTransposeOrAdjoint,
```

Comment:
Let's split this into two flags `bool isTransposeATranspose` and `bool isTransposeAAdjoint`, to be consistent with `inferFftOp`.

Reply:
We won't need these flags in the latest commit